### PR TITLE
Add some additional stats for debugging edit lags

### PR DESCRIPTION
**Pull request by ZappoMan**
* adds stats for the outbound edit queue
* improves the server error message for failed edit
* fixes a bug in calculated average updates per second
* added packet sequence stats to entity server for inbound edit packets

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 923e1ed208936788deeb33e98bf4bf699e715a56 available at: https://gist.github.com/2e2f8b8e7705a6af081c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6090.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6090.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 82a211174bf4fc361b75c7dc10102d95ccca4449 available at: https://gist.github.com/2e2f8b8e7705a6af081c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6090.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6090.dmg

File: assignment-client/src/octree/OctreeServer.cpp [hunk 1]
```diff
@@ -639,11 +641,16 @@ bool OctreeServer::handleHTTPRequest(HTTPConnection* connection, const QUrl& url
...
 
         float averageElementsPerPacket = totalPacketsProcessed == 0 ? 0 : (float)totalElementsProcessed / totalPacketsProcessed;
 
-        statsString += QString("   Current Inbound Packets Queue: %1 packets\r\n")
+        statsString += QString("   Current Inbound Packets Queue: %1 packets \r\n")
             .arg(locale.toString((uint)currentPacketsInQueue).rightJustified(COLUMN_WIDTH, ' '));
+        statsString += QString("        Packets Queue Network IN: %1 PPS \r\n")
+            .arg(locale.toString(incomingPPS, 'f', FLOAT_PRECISION).rightJustified(COLUMN_WIDTH, ' '));
+        statsString += QString("    Packets Queue Processing OUT: %1 PPS \r\n")
+            .arg(locale.toString(incomingPPS, 'f', FLOAT_PRECISION).rightJustified(COLUMN_WIDTH, ' '));
```
Comment (birarda) [inline assignment-client/src/octree/OctreeServer.cpp]: these are the same stat `incomingPPS`

**Comment by hifi-gustavo** [comment]
Build console output for commit: e1570e319737335e991f034e36752734f3b469dd available at: https://gist.github.com/2e2f8b8e7705a6af081c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6090.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6090.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3718149157cb82eecf92620b07c0c182e118fd41 available at: https://gist.github.com/2e2f8b8e7705a6af081c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6090.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6090.dmg

**Pull request merged**